rpcd/playbooks/setup-logging.yml

```diff
@@ -14,6 +14,7 @@
 # limitations under the License.
 
 - include: elasticsearch.yml
+  when: "{{ lookup('env', 'UPGRADE_ELASTICSEARCH')}}" == 'no'
```

Comment:
Travis job isn't too happy with this lookup: https://travis-ci.org/rcbops/rpc-openstack/jobs/292866233#L762